tests/frontend-storage/src/backend/server-storage-proxy/Server.ts

```diff
@@ -0,0 +1,138 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as path from "path";
+
+import * as express from "express";
+import { Container } from "inversify";
+
+import { Bindable } from "@itwin/cloud-agnostic-core";
+import {
+  ServerStorage,
+  ServerStorageDependency,
+} from "@itwin/object-storage-core";
+
+import * as Common from "./Common";
+
+export class ServerStorageProxyBackend extends Bindable {
+  public readonly container = new Container();
+
+  constructor() {
+    super();
+    this.requireDependency(ServerStorageDependency.dependencyType);
+  }
+
+  public start(config: { port: number }): void {
+    this.bindDependencies(this.container);
+    const serverStorage = this.container.get(ServerStorage);
+
+    const app = express();
+    const publicDir = path.resolve(__dirname, "..", "..", "public");
+    app.use(express.static(publicDir));
+    app.use(express.json());
+
+    // Not exactly REST but it's for tests only
```

Comment:
I don't think the comment is necessary - it does not really add information, it only justifies the design choice.